src/replit_river/client.py

```diff
@@ -105,7 +104,7 @@ async def send_upload(
         init_serializer: Optional[Callable[[InitType], Any]],
         request_serializer: Callable[[RequestType], Any],
         response_deserializer: Callable[[Any], ResponseType],
-        error_deserializer: Callable[[Any], ErrorType],
```

Comment:
Can you revert these changes in #141? This looks like it's necessary for now, but once the other issue is fixed (`Unknown` hiding the type incompatibility) we want to have fewer `Any`'s